M2/Macaulay2/packages/Colon.m2

```diff
@@ -0,0 +1,885 @@
+---------------------------------------------------------------------------
+-- PURPOSE : Computation of quotient, saturation, and annihilator
+--
+-- UPDATE HISTORY : created 14 April 2018 at M2@UW;
+--                  updated November 2020
+--
+-- TODO : 1. why are there shadowed symbols?
+--        2. any strategies to incorporate from Shimoyama-Yokoyama.m2
+--           or greedySat in GTZ.m2 or getSaturation in newGTZ.m2
+--        3. which algorithms for Ideals can be adapted to submodules?
+--        4. move radical here, perhaps intersect as well
+--        5. employ storefun! Maybe call it declare
+---------------------------------------------------------------------------
+newPackage(
+    "Colon",
```

Comment:
Having worked on it a lot, I'm rather fond of this name. I think "IdealQuotients" is a bad name because this package does a lot more than that. Moreover, any name with "Quotient" will be confusing. If you insist, I could get behind "Saturation", but I'll defer to @mikestillman here.

Reply:
Why not have several packages, then?  One for ideal quotients, one for saturation, etc.?

Reply:
It's too late to change this decision.